client/components/jetpack/daily-backup-status/status-card/backup-last-failed.tsx

```diff
@@ -0,0 +1,45 @@
+import { Gridicon } from '@automattic/components';
+import { ExternalLink } from '@wordpress/components';
+import { useTranslate } from 'i18n-calypso';
+import { FunctionComponent } from 'react';
+import { recordTracksEvent } from 'calypso/state/analytics/actions/record';
+import type { Moment } from 'moment';
+
+type Props = {
+	baseBackupDate: Moment;
+	eventsCount: number;
+	selectedBackupDate: Moment;
+	learnMoreUrl?: string;
+};
+
+export const BackupLastFailed: FunctionComponent< Props > = () => {
+	const translate = useTranslate();
+	const learnMoreLink = 'https://jetpack.com/support/backup/troubleshooting-jetpack-backup/';
+
+	return (
+		<div className="backup-last-failed__wrapper">
+			<span className="backup-last-failed__info">
+				<Gridicon icon="notice-outline" />
+				{ translate(
+					"We encountered some issues with the latest backup, but don't worry! " +
+						"We're using the most recent backup available. " +
+						'{{ExternalLink}}Learn more.{{/ExternalLink}}',
+					{
+						components: {
+							ExternalLink: (
+								<ExternalLink
+									href={ learnMoreLink }
+									children={ null }
+									rel="noopener noreferrer"
+									onClick={ () =>
+										recordTracksEvent( 'calypso_jetpack_backup_last_faileds_learn_more_click' )
```

Comment:
You might need to call this function using the useDispatch() hook, like this: https://github.com/Automattic/wp-calypso/blob/6f55caa350b3bc6fb1172b7ca24df94e029c018c/client/components/jetpack/daily-backup-status/action-buttons.jsx#L28

Reply:
Also, could we move this `onClick` handler into a separate function? This will avoid inline function creation on each render.

Reply:
Done!